packages/notifi-graphql/lib/NotifiEventEmitter.ts

```diff
@@ -1,15 +1,31 @@
 import { Client as WebSocketClient } from 'graphql-ws';
+import { StateChangedEvent, TenantEntityChangeEvent } from './gql/generated';
 
-export type NotifiEmitterEvents = {
+/** IMPORTANT: the guidelines to remove the event listener:
+ * 1. For status events (NotifiWebSocketStatusEvents or NotifiSubscriptionStatusEvents), calling removeEventListener will remove the listener.
+ * 2. For graphql subscription events (NotifiSubscriptionEvents), calling removeEventListener will not unsubscribe the subscription. additionally, you need to call subscription.unsubscribe() to remove the subscription.
+ */
```

Comment:
Important note for unsubscribe & remove gql subscription event listener CC: @nimesh-notifi